README.md

```diff
@@ -65,11 +65,16 @@ $ mvn install
   downloaded data:
 
 ```
-$ mvn exec:java
-$ cd ..
+$ java -jar target/platform-datacleaner-jar-with-dependencies.jar
+```
+
+* Next, start an instance of a HillviewServer that listens on port 3569.
+
+```
+$ java -jar target/platform-jar-with-dependencies.jar 127.0.0.1:3569
```

Comment:
Let's simplify the number of steps to run the demo as much as possible. Maybe we can just script the launch of the service and the web server.